modules/otel-config-instrumentation.adoc

```diff
@@ -98,6 +98,105 @@ spec:
 
 |===
 
+[id="otel-instrumentation-options_exporter_{context}"]
+== Exporter configuration
+
+Although the `Instrumentation` custom resource supports setting up one or more exporters per signal, auto-instrumentation configures only the OTLP Exporter. So you must configure the endpoint to point to the OTLP Receiver on the Collector.
+
+.Sample exporter TLS CA configuration using a config map
+[source,yaml]
+----
+# ...
+spec
+  exporter:
+    endpoint: https://production-collector.observability.svc.cluster.local:4317  # <1>
+    tls:
+      configMapName: ca-bundle  # <2>
+      ca_file: service-ca.crt # <3>
+# ...
+----
+<1> Specifies the OpenTelemetry Protocol (OTLP) endpoint using the HTTPS scheme and TLS.
+<2> Specifies the name of the config map. The config map must already exist in the namespace of the pod injecting the auto-instrumentation.
+<3> Points to the CA certificate in the config map or the absolute path to the certificate if the certificate is already present in the workload file system.
+
+.Sample exporter mTLS configuration using a Secret
+[source,yaml]
+----
+# ...
+spec
+  exporter:
+    endpoint: https://production-collector.observability.svc.cluster.local:4317  # <1>
+    tls:
+      secretName: serving-certs # <2>
+      ca_file: service-ca.crt # <3>
+      cert_file: tls.crt # <4>
+      key_file: tls.key # <5>
+# ...
+----
+<1> Specifies the OpenTelemetry Protocol (OTLP) endpoint using the HTTPS scheme and TLS.
+<2> Specifies the name of the Secret for the `ca_file`, `cert_file`, and `key_file` values. The Secret must already exist in the namespace of the pod injecting the auto-instrumentation.
+<3> Points to the CA certificate in the Secret or the absolute path to the certificate if the certificate is already present in the workload file system.
+<4> Points to the client certificate in the Secret or the absolute path to the certificate if the certificate is already present in the workload file system.
+<5> Points to the client key in the Secret or the absolute path to a key if the key is already present in the workload file system.
+
+[NOTE]
+====
+A config map has higher precedence than a Secret.
```

Comment:
This could be more explanatory. e.g.: The `ca_file` can be provided from config map and secret. If both are specified the CA from the config map has a higher precedence.

Reply:
Fixed.